rentalPrice.js

```diff
@@ -1,33 +1,27 @@
-// age - age of driver
-// licence - number of full years person holds driving licence
-// clazz - class of the car from 1 (smallest) to 5 (largest) that person wishes to rent
-// acc - has s/he caused any accidents within last year
-// acc2 - has s/he participated (but not caused) in any accidents within last year
-// season - if it is high season or not
-function price(age, licence, clazz, acc, acc2, season) {
+function price(driverAge, licenceAge, vehicleClass, accidentsCaused, accidentsParticipated, highSeason) {
```

Comment:
accidentsParticipated - is this variable used anywhere? is it needed? are we missing some business rule check?